src/connection-manager/connections.ts

```diff
@@ -45,15 +56,79 @@ function subscribe(ws: WebSocket): void {
 }
 
 /**
- * Handles a WebSocket message received.
+ * Sends a ledger_entry WebSocket request to retrieve amendments enabled on a network.
+ *
+ * @param ws - A WebSocket object.
+ */
+function getAmendmentLedgerEntry(ws: WebSocket): void {
+  ws.send(
```

Comment:
Perhaps all of these `ws.send` functions could go in a different file, to satisfy the linter?